tests/KafkaFlow.IntegrationTests/Core/Bootstrapper.cs

```diff
@@ -31,19 +30,19 @@ internal static class Bootstrapper
         internal const string AvroGroupId = "consumer-avro";
         internal const string JsonGroupId = "consumer-json";
 
-        private const string ProtobufTopicName = "test-protobuf";
-        private const string ProtobufSchemaRegistryTopicName = "test-protobuf-sr";
-        private const string JsonSchemaRegistryTopicName = "test-json-sr";
-        private const string JsonTopicName = "test-json";
-        private const string GzipTopicName = "test-gzip";
-        private const string JsonGzipTopicName = "test-json-gzip";
-        private const string ProtobufGzipTopicName = "test-protobuf-gzip";
-        private const string ProtobufGzipTopicName2 = "test-protobuf-gzip-2";
-        private const string AvroTopicName = "test-avro";
+        private const string _protobufTopicName = "test-protobuf";
```

Comment:
Constants should use PascalCase.

Check the [C# identifier naming rules and conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names)

> Use PascalCase for constant names, both fields and local constants.

You can also check examples of the recent Aspire project where this convention is used:
https://github.com/search?q=repo%3Adotnet%2Faspire%20private%20const%20string&type=code

Reply:
Done. Please check.